src/functions.php

```diff
@@ -43,11 +43,12 @@ function captureException(\Throwable $exception): ?EventId
 /**
  * Captures a new event using the provided data.
  *
- * @param array<string, mixed> $payload The data of the event being captured
+ * @param Event                  $event The event being captured
+ * @param \Sentry\EventHint|null $hint  May contain additional information about the original exception
  */
-function captureEvent(array $payload): ?EventId
+function captureEvent(Event $event, ?EventHint $hint = null): ?EventId
```

Comment:
@HazAT It looks like some of the SDKs are accepting the `$hint` parameter (e.g. Python) and some aren't (e.g. JS). I have nothing cons this change, just want to be sure that we are doing well

Reply:
This is good!